src/main/java/mate/project/repository/BookRepository.java

```diff
@@ -7,4 +7,6 @@ public interface BookRepository {
     Book save(Book book);
 
     List<Book> findAll();
+
+    Book getBookById(Long id);
```

Comment:
You should use Optional here:
```suggestion
    Optional<Book> getBookById(Long id);
```

Reply:
Okay, I went by the video where the coach explained to use optional for methods like findBookById where we not sure if the book exists, and for methods with get we dont need to. But I see and understand why it is better to use optional for both cases